### DISCUSSION TITLE
Painted Faces - Possibly a training bug?

### DISCUSSION THREAD

**Discussion by XetalSL** [Acly/krita-ai-diffusion#548] [General]
With this face (which was generated by Stable Diffusion):
![face](https://github.com/Acly/krita-ai-diffusion/assets/155661216/080b888a-75ed-4aea-97de-50e4f8185958)

Example Prompt:

```
british 25 year old female model in minimalistic formal attire. portrait.
High precision, high detail, realistic, no eye shadow, sharp, focus, documentary, in manor.
```

Used with the face control (any % strength) and the Digital Artwork trained set and no LoRA's loaded, I seam to get generations with faces being splotchy or painted faces: 

![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/170a600e-219c-41e5-b38c-873c8b88f1e5)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/e045e738-46b6-49e8-a465-2b45a2e022f4)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/73ba622f-7977-4d16-8c23-2dc83a36880f)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/5b33482b-f4ba-426c-85ec-bc16a9b1bd40)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/803e6256-601a-4a58-b3a1-3d3b4e2c1702)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/80fdd929-4910-4b2b-91d1-bf3e923b9fe5)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/f62a9525-5212-41f2-bb24-e21ac233afba)
![image](https://github.com/Acly/krita-ai-diffusion/assets/155661216/246389d8-5d44-4bb6-8d81-01e96ae29bd6)

**Comment by ETCore7** [comment]
I can't reproduce your results. Using your original as face reference just inpaints the face on target pictures.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/79119bf3-d632-48c1-ba11-7eb3f6522554)

**Reply by XetalSL** [reply]
I'm not in painting - just generating from scratch.

**Comment by ETCore7** [comment]
I still can't reproduce it, tried several default models.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/03f9bf53-f581-4132-81b4-4b6ed6e05ff1)

**Comment by ETCore7** [comment]
You may have changed the sampler to DPM 2M SDE Karras, default is DPM 2M Karras (see below).

![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/71d3b0ab-75c0-408d-bd25-be1d89c5db93)
_DPM 2M SDE Karras_

![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/45e0445b-8d30-48e9-afe7-0a6ad02219c0)
_Default_

**Reply by XetalSL** [reply]
uhuh! - i did change that - I've tried it out and i get a lot less weirdness. out of 10 generations 1 was funky.

**Comment by ETCore7** [comment]
(edit)

Anyway, both samplers work fine with their default settings without controlnets, but if you want to use DPM 2M SDE Karras (= default sampler) with the Face controlnet you must lower its Guidance Strength to i.e. 2.0.

**Comment by ETCore7** [comment]
PS, if restoring defaults via the button results in a crash too, here are the default settings:

![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/96020001-356f-40b5-bc4e-7f152a3d2ea7)
